hooks/invocation/invocation.go

```diff
@@ -0,0 +1,35 @@
+package invocation
+
+import "time"
+
+type Action int
+
+const (
+	Nop Action = iota
+	Update
+	Reject
```

Comment:
I'm curious what you plan to use the Action code for? We will know if no action is taken if there are no mutations and from today's meeting we expect to pass back a NBR (no bid response) code is it's rejected.

Reply:
We tried to implement this as similar as possible to the PBS-Java code, but I agree with you, we do not need the Update/Nop actions, this can be concluded from the presence of mutations.
Rejection itself we will implement in the following tasks.